dora/notifications/tasks/structures.py

```diff
@@ -60,15 +60,14 @@ def task_type(cls):
 
     @classmethod
     def candidates(cls):
-        # liste des structures sans membres enregistrés,
-        # avec au moins un admin en attente d'inscription
-        # depuis un mois
+        # structures sans services, hors agences FT
+        # avec au moins un admin inscrit depuis un mois
         one_month_ago = timezone.now() - relativedelta(months=1)
         return Structure.objects.filter(
             services=None,
-            membership=None,
-            putative_membership__is_admin=True,
-            putative_membership__creation_date__lt=one_month_ago,
+            code_safir_pe=None,
```

Comment:
nit : le plus fiable me semble de tester que le code siren est différent de SIREN_POLE_EMPLOI (dora.core.constants).
On utilise ça à plusieurs endroit: `not siret.startswith(SIREN_POLE_EMPLOI)`

Reply:
Merci, je n'avais pas vu ça encore. Je corrige ça.